packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -26,8 +26,11 @@
   "force_debugger_stop_text": "SFDX: Stop Apex Debugger Session",
   "force_config_list_text": "SFDX: List All Config Variables",
   "force_alias_list_text": "SFDX: List All Aliases",
+  "force_org_delete_default_text": "SFDX: Delete Default Org",
```

Comment:
@ekapner these are new commands in the command palette

Reply:
@lcampos - is this more like Remove This Org as the Default (gah, this is a hard one to convey)? We're not actually deleting it? Like poof?

Reply:
@ekapner we are running force:org:delete & force:org:list --clean for these ones (https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_org.htm). If they select a scratch org or sandbox it will be deleted